### PR TITLE
loss issue in RegL1Loss

### PR DESCRIPTION
**Pull request by JavisPeng**
```python
def forward(self, output, mask, ind, target):
    pred = _transpose_and_gather_feat(output, ind)
    mask = mask.unsqueeze(2).expand_as(pred).float()
    # loss = F.l1_loss(pred * mask, target * mask, reduction='elementwise_mean')
    loss = F.l1_loss(pred * mask, target * mask, size_average=False)
    loss = loss / (mask.sum() + 1e-4)
    return loss
```
since the mask has expanded, the number of value 1 should be twice the previous, which results in loss=loss*0.5

### CODE REVIEW

**Comment by xingyizhou** [comment]
Thank you for your contribution. However this will impact the performance and needs further verification. Closing it for now.
